ci/ray_ci/automation/weekly_green_metric.py

```diff
@@ -39,6 +47,11 @@ def main(production: bool) -> None:
         )
         logger.info("Weekly green metric updated successfully")
 
+    if check and blockers.totalCount != 0:
+        raise ValueError(
```

Comment:
could you just raise `Exception`, or maybe just print a line to stderr, and `sys.exit(1)`? `ValueError` means an argument has inappropriate value.

Reply:
done!